README.md

```diff
@@ -1 +1,148 @@
-# todo-list-es6
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+ <h3><b>Microverse</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🚀 Visit Site](#website)
+  - [🎥 Walkthrough](#walkthrough)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ToDo List] <a name="about-project"></a>
+
+This project portrays a todo taking application
+
+**[todo-lists ]** is an html,css, js project that helps a user take to dos.
+
+## 🚀 VISIT SITE <a name="website"></a>
+
+<a name="<https://king-benx.github.io/todo-list-es6/>">https://king-benx.github.io/todo-list-es6/</a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML, CSS AND JAVASCRIPT  
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML, CSS & JS</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- An HTML File
+- A CSS File
+- JavaScript file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```git clone git@github.com:King-Benx/todo-list.git```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+1. Right click index.html file and run in browser.
+2. Open index file in vscode right click and open in live server
```

Comment:
- [ ] Nice work on this section. however, you didn't include how to create and build your app using the webpack on your project. Kindly add the steps to create a build for your project